trieste/models/gpflux/models.py

```diff
@@ -23,13 +23,25 @@
 
 from ...data import Dataset
 from ...types import TensorType
-from ..interfaces import TrainableProbabilisticModel
+from ..interfaces import (
+    HasReparamSampler,
+    HasTrajectorySampler,
+    ReparametrizationSampler,
+    TrainableProbabilisticModel,
+    TrajectorySampler,
+)
 from ..optimizer import BatchOptimizer
 from .interface import GPfluxPredictor
-from .utils import sample_dgp
+from .sampler import (
+    DeepGaussianProcessReparamSampler,
+    DeepGaussianProcessTrajectorySampler,
+    sample_dgp,
```

Comment:
you should import these in `trieste/models/gpflux/__init__.py` as well no?

Reply:
I'm not entirely sure as they are quite specific to the `gpflux` stuff, do we want the user to be able to access them outside more easily?

Reply:
if they won't be needed outside, then sure